src/main/resources/matlab-bamboo-plugin.properties

```diff
@@ -5,14 +5,14 @@ matlab.plugin.command=Command
 plugin.command.error=Command field is empty.
 
 matlab.helpPath = Full path to MATLAB root folder
-matlab.options = Specify startup options
-matlab.options.field = Options
+matlab.options = Startup options
 
 # MATLAB Build keys
-matlab.build.tasks = Build tasks
+matlab.build.tasks = Tasks
+matlab.build.options = Build options
```

Comment:
Will the "Specify build options" check box appear below the "Tasks" box?

Reply:
Yes, adding screenshot for clarity